.github/workflows/test.yml

```diff
@@ -16,8 +16,14 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: '18'
+      - name: Setup Deno
+        if: matrix.package-manager == 'deno'
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
```

Comment:
Ah, We need to use latest deno version

https://github.com/denoland/setup-deno